File: plugin/src/main/java/ru/arc/auction/AuctionItemDto.java
package ru.arc.auction;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.ArrayList;
import java.util.List;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class AuctionItemDto {
    String display;
    String seller;
    String price;
    long expire;
    String category;
    int amount;
    int priority;
    String uuid;
    boolean exist;
    List<String> lore = new ArrayList<>();
}

File: src/main/kotlin/ru/arc/auction/AuctionItemDto.kt
package ru.arc.auction

data class AuctionItemDto(
    @JvmField var display: String? = null,
    @JvmField var seller: String? = null,
    @JvmField var price: String? = null,
    @JvmField var expire: Long = 0,
    @JvmField var category: String? = null,
    @JvmField var amount: Int = 0,
    @JvmField var priority: Int = 0,
    @JvmField var uuid: String? = null,
    @JvmField var exist: Boolean = false,
    @JvmField var lore: MutableList<String> = ArrayList(),
)
